packages/now-python/now_handler.py

```diff
@@ -12,12 +12,20 @@ def now_handler(event, context):
     path = payload['path']
     headers = payload['headers']
     method = payload['method']
-    
-    res = requests.request(method, 'http://0.0.0.0:3000' + path, headers=headers)
+    encoding = payload.get("encoding")
+    body = payload['body']
+
+    if (
+        (body is not None and len(body) > 0) and
+        (encoding is not None and encoding == "base64")
+    ):
+        body = base64.b64decode(body)
+
+    res = requests.request(method, 'http://0.0.0.0:3000' + path,
+            headers=headers, data=body)
```

Comment:
```suggestion
            headers=headers, data=body, allow_redirects=False)
```

It's probably a better idea to disable server-side redirection and leave the response (e.g. 302 Found) as it is, or else the redirection will trigger an error in Now's execution environment.

See kenneth-reitz/now-builders#1 for details.

Reply:
I'm good with this change if the team approves

Reply:
Makes sense to me 👍